src/UseCases/ExtractAddress/RejectedAddressWriter.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\OtrsExtractAddress\UseCases\ExtractAddress;
+
+use WMDE\OtrsExtractAddress\Domain\SourceData;
+
+/**
+ * @license GNU GPL v2+
+ * @author Gabriel Birke < gabriel.birke@wikimedia.de >
+ */
+class RejectedAddressWriter {
+
+	use CSVFormatter;
+
+	private $outputStream;
+
+	public function __construct( $outputStream ) {
```

Comment:
Nope, resources (DB resource, file handler resource), can't be type hinted yet. But I made another commit where I replace the resource with `SplFileObject`.